boltzmann/class/class_interface.py

```diff
@@ -253,8 +293,15 @@ def execute(block, config):
         params = get_class_inputs(block, config)
         c.set(params)
 
-        # Run calculations
-        c.compute()
+        # saves neff if we are using nur, for cases where nur is sampled:
+        block[cosmo, 'nnu'] = c.Neff()
+
+        if not config['background_only']:
+            # Run perturbation calculations
+            c.compute()
+        else:
```

Comment:
This `else` branch is unnecessary. You can just put the comment in the one above